plugin/trino-exchange-filesystem/src/test/java/io/trino/plugin/exchange/filesystem/local/TestLocalFileSystemExchangeManager.java

```diff
@@ -26,10 +30,22 @@ protected ExchangeManager createExchangeManager()
     {
         String baseDirectory1 = System.getProperty("java.io.tmpdir") + "/trino-local-file-system-exchange-manager-1";
         String baseDirectory2 = System.getProperty("java.io.tmpdir") + "/trino-local-file-system-exchange-manager-2";
-        return new FileSystemExchangeManagerFactory().create(ImmutableMap.of(
-                "exchange.base-directories", baseDirectory1 + "," + baseDirectory2,
-                // to trigger file split in some tests
-                "exchange.sink-max-file-size", "16MB",
-                "exchange.source-handle-target-data-size", "1MB"));
+        return new FileSystemExchangeManagerFactory().create(
+                ImmutableMap.of(
+                        "exchange.base-directories", baseDirectory1 + "," + baseDirectory2,
+                        // to trigger file split in some tests
+                        "exchange.sink-max-file-size", "16MB",
+                        "exchange.source-handle-target-data-size", "1MB"),
+                new ExchangeManagerContext() {
```

Comment:
Please create `TestingExchangeManagerContext` and reuse

Reply:
done